docs/rules/terraform_no_short_circuit_evaluation.md

```diff
@@ -0,0 +1,60 @@
+# terraform_no_short_circuit_evaluation
```

Comment:
`terraform_logical_short_circuit` would be a better name. Most of the other rules are anti-patterns but none uses a `no` prefix.